NYTimes.com.js

```diff
@@ -165,8 +166,8 @@ function scrape(doc, url) {
 	if (metaTags["keywords"]) {
 		var keywords = metaTags["keywords"];
 		newItem.tags = keywords.split(",");
-		for (var i in newItem.tags) {
-			newItem.tags[i] = newItem.tags[i].replace("  ", ", ");
+		for (var t in newItem.tags) {
```

Comment:
No, `var` for both. (Doesn't really matter in translators, since they're in a sandbox, but otherwise if the first one gets removed the second becomes a global assignment, which would trigger an error in strict mode.)